examples/smac/feedforward/decentralised/run_ippo_smac_death_masking.py

```diff
@@ -0,0 +1,107 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Run IPPO on SMAC and allowing death masking."""
+
+
+import functools
+from datetime import datetime
+from typing import Any
+
+import optax
+from absl import app, flags
+
+from mava.systems import ippo
+from mava.utils.environments.smac_utils import make_environment
+from mava.utils.loggers import logger_utils
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    "map_name",
+    "3m",
+    "Starcraft 2 micromanagement map name (str).",
+)
+
+flags.DEFINE_string(
+    "mava_id",
+    str(datetime.now()),
+    "Experiment identifier that can be used to continue experiments.",
+)
+flags.DEFINE_string("base_dir", "~/mava", "Base dir to store experiments.")
+
+
+def main(_: Any) -> None:
+    """Example running feedforward MADQN on SMAC environment."""
+
+    # Environment
+    environment_factory = functools.partial(
+        make_environment, map_name=FLAGS.map_name, death_masking=True
+    )
+
+    # Networks.
+    def network_factory(*args: Any, **kwargs: Any) -> Any:
+        return ippo.make_default_networks(  # type: ignore
+            policy_layer_sizes=(256, 256, 256),
```

Comment:
Can we update these as in the other PR, where we use smaller networks as recommended by the paper.

Reply:
Yeah sure but the benchmark is on this network size, should we repeat the benchmarking?